### PR TITLE
Some style fixes

### PR DESCRIPTION
**Pull request by willtebbutt**
I picked this file at random, and have made some fixes for conformity with our style guide. There shouldn't be _any_ changes to functionality here, it's all superficial.

### CODE REVIEW

**Comment by yebai** [comment]
Thanks @willtebbut, looks good to me!

**Comment by xukai92** [comment]
I will take another look soon.

File: src/trajectory.jl [hunk 1]
```diff
@@ -146,25 +149,28 @@ transition(
...
 ###
 
-struct StaticTrajectory{S<:AbstractTrajectorySampler,I<:AbstractIntegrator} <: AbstractTrajectory{I}
+struct StaticTrajectory{S<:AbstractTrajectorySampler, I<:AbstractIntegrator} <: AbstractTrajectory{I}
     integrator  ::  I
     n_steps     ::  Int
 end
 
-Base.show(io::IO, τ::StaticTrajectory{S,I}) where {I,S<:EndPointTS} =
+function Base.show(io::IO, τ::StaticTrajectory)
```
Comment (xukai92) [inline src/trajectory.jl]: We do need the type info for correct dispatch here.

File: src/trajectory.jl [hunk 2]
```diff
@@ -146,25 +149,28 @@ transition(
...
     print(io, "StaticTrajectory{EndPointTS}(integrator=$(τ.integrator), λ=$(τ.n_steps)))")
-Base.show(io::IO, τ::StaticTrajectory{S,I}) where {I,S<:MultinomialTS} =
+end
+
+function Base.show(io::IO, τ::StaticTrajectory)
```
Comment (xukai92) [inline src/trajectory.jl]: Again we do need the type info for correct dispatch here.

File: src/trajectory.jl (hunk 1 above)
Comment (xukai92) [reply src/trajectory.jl]: Maybe you are intended to do what you did for `HMCDA`?

File: src/trajectory.jl [hunk 3]
```diff
@@ -334,7 +347,7 @@ Create an instance for the No-U-Turn sampling algorithm.
 function NUTS{S,C}(
     integrator::I,
     max_depth::Int=10,
-    Δ_max::F=1000.0
+    Δ_max::F=1000.0,
```
Comment (xukai92) [inline src/trajectory.jl]: @yebai We actually have a guide on always including "the trailing comma when working with expanded arrays, tuples or functions notation".

File: src/trajectory.jl (hunk 1 above)
Comment (willtebbutt) [reply src/trajectory.jl]:
```suggestion
function Base.show(io::IO, τ::StaticTrajectory{<:EndPointTS})
```

File: src/trajectory.jl (hunk 2 above)
Comment (willtebbutt) [reply src/trajectory.jl]:
```suggestion
function Base.show(io::IO, τ::StaticTrajectory{<:MultinomialTS})
```

**Comment by willtebbutt** [comment]
Good catches @xukai92

**Comment by xukai92** [comment]
Thanks for correcting them so quickly! I will merge the PR once the CI passes.

**Pull request merged**